Jenkinsfile

```diff
@@ -21,11 +21,11 @@ properties([buildDiscarder(logRotator(artifactNumToKeepStr: '5', numToKeepStr: e
 
 def buildOs = 'linux'
 def buildJdk = '17'
-def buildMvn = '3.8.x'
+def buildMvn = '4.0.0-rc-2'
 def runITsOses = ['linux']
 def runITsJdks = ['17', '21']
-def runITsMvn = '3.8.x'
-def runITscommand = "mvn clean install -Prun-its,embedded -B -U -V" // -DmavenDistro=... -Dmaven.test.failure.ignore=true
+def runITsMvn = '4.0.0-rc-2'
```

Comment:
I'd make this as GH CI does: run IT build _and_ run ITs with "just built" master maven instead